pymathics/natlang/main.py

```diff
@@ -1203,8 +1222,8 @@ def _property(
         except MessageException as e:
             e.message(evaluation)
 
-    def eval(self, word, evaluation: Evaluation, options: dict):
-        "WordData[word_, OptionsPattern[%(name)s]]"
+    def eval(self, word, evaluation: Evaluation, options: dict) -> Optional[Expression]:
+        "WordData[word_, OptionsPattern[WordData]]"
         if word.get_head_name() == "System`StringExpression":
             return Expression(SymbolDictionaryLookup, word)
         elif isinstance(word, String) or word.get_head_name() == "System`List":
```

Comment:
Maybe `get_head` here and above?

Reply:
This and other places have now been corrected.